app/assets/stylesheets/application.scss

```diff
@@ -1,9 +1,12 @@
+/* variables */
+@import 'bootstrap-variables'; // our overwrites
+@import 'bootstrap/variables'; // the original bootstrap ones
```

Comment:
Bootstrap variables [are defined here](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss) with `!default` so the values will be used from `bootstrap/variables` unless they are defined before.